user-docs/SubmittingJobs.md

```diff
@@ -0,0 +1,133 @@
+You will right away see a page with all AWS repositories.
+Create a new repository with a meaningful name, it's good practice to include your user ID at the start of the repository name.
+Note the URL that AWS gives you, you will need it later.
+
+When your AWS repository is created you should login to it.
+If you don't have the AWS CLI tool yet, you can install it by doing `pip install awscli` and `aws configure` to enter your credentials.
+The credentials you need can be created within AWS as explained on [this documentation page](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+After having installed AWS CLI, run the following command:
+
+```
+aws ecr get-login --no-include-email --region eu-west-1
+```
+and run the command that was given in the output of the console.
+For DockerHub, you can authenticate using `docker login`.
+
+If you have trouble creating an AWS Docker Registry you can request one of the administrators (hyperion@bigdatarepublic.nl) to create one for you and guide you through the setup process.
+
+Once you have a repository up and running, you can tag and push your Docker image like so:
+
+```
+docker tag <image_id_or_local_name> 0123456789012.dkr.ecr.eu-west-1.amazonaws.com/awesome-project-trainer:latest
+docker push 0123456789012.dkr.ecr.eu-west-1.amazonaws.com/awesome-project-trainer:latest
+```
+
+We understand that this process is quite cumbersome right now.
+In the future, this whole process will be automated through the [hyperion-cli](https://github.com/BigDataRepublic/hyperion-cli).
+
+## Uploading data to the cluster
+Make sure to read [this guide](http://github.com/BigDataRepublic/hyperion/blob/master/user-docs/GettingDataOnCluster.md) on how to get data on the cluster.
+Also make sure to remember the path where you store the data (either `/home/<user>/<project>` or `/home/<user>/scratch/<project>`).
+
+## Creating a Kubernetes deployment
+To submit jobs to Kubernetes, you will need to create a deployment file referring to the image to run, the required resources and the necessary data folders to mount.
+
+An example file is shown below, which you should save as `deployment.yaml`
+
+```
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: nvidia-test  # the name of your job
+spec:
+  template:
+    spec:
+      volumes:
+      - hostPath:
+          path: /home/sreitsma/test  # the folder you want to mount into the container
+          type: Directory
+        name: test
+      containers:
+      - name: awesome-project-trainer  # give the image in your job a name, can be anything
+        image: nvidia/cuda  # the path to the image, can be hosted on DockerHub or on our AWS ECR repositories
+        command: ["nvidia-smi"]
+        resources:
+          limits:
+            nvidia.com/gpu: 1  # request a single GPU
+            memory: "4096Mi"  # the maximum amount of RAM your application can/will use

```

Comment:
Maybe make a set of recommended settings and limits?

Reply:
Good idea, added! @DJLemkes